src/main/java/de/zalando/aruha/nakadi/validation/RFC3339DateTimeValidator.java

```diff
@@ -0,0 +1,39 @@
+package de.zalando.aruha.nakadi.validation;
+
+import org.everit.json.schema.FormatValidator;
+
+import java.time.format.DateTimeParseException;
+import java.util.Optional;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static java.time.format.DateTimeFormatter.ISO_OFFSET_DATE_TIME;
+
+public class RFC3339DateTimeValidator implements FormatValidator {
+
+    private final String errorMessage = "must be a valid date-time";
+
+    // Valid offsets are either Z or hh:mm. The format hh:mm:ss is not valid
+    private final String dateTimeOffsetPattern = "^.*(Z|((\\+|-)\\d\\d:\\d\\d))$";
+    private final Pattern pattern = Pattern.compile(dateTimeOffsetPattern);
+    private final Optional<String> error = Optional.of(errorMessage);
+
+    @Override
+    public Optional<String> validate(final String dateTime) {
+        try {
+            java.time.LocalDate.parse(dateTime, ISO_OFFSET_DATE_TIME);
```

Comment:
Done in https://github.com/zalando/nakadi/pull/355/commits/b740c11e371f4ae0bae52f49928e1701880999e7